core/frontend/src/Viewport.ts

```diff
@@ -262,6 +262,14 @@ export interface ReadPixelsArgs {
   excludedElements?: Iterable<Id64String>;
 }
 
+/** Arguments supplied to [[Viewport.readImageToCanvas]].
+ * @public
+ */
+export interface ReadImageToCanvasOptions {
+  /** If true, canvas decorations will be included in the saved image. */
+  omitCanvasDecorations: boolean;
```

Comment:
```suggestion
  omitCanvasDecorations?: boolean;
```
Don't require people to specify this. Default false.